lib/assets/stylesheets/_govuk_tech_docs.scss

```diff
@@ -1,6 +1,23 @@
 $govuk-assets-path: "/assets/govuk/assets/";
 
-@import "govuk/all";
+// Include only the bits of GOV.UK Frontend we need
+@import "govuk/settings/all";
+@import "govuk/tools/all";
+@import "govuk/helpers/all";
+
+@import "govuk/core/all";
+@import "govuk/objects/all";
+
+@import "govuk/components/footer/footer";
+@import "govuk/components/header/header";
+@import "govuk/components/inset-text/inset-text";
+@import "govuk/components/input/input";
+@import "govuk/components/tag/tag";
+@import "govuk/components/skip-link/skip-link";
+@import "govuk/components/warning-text/warning-text";
```

Comment:
When I've done this sort of thing in the past I've left in the full component list as comments for other developers ease of use but probably is an opinionated idea.

Reply:
I'm going to leave as-is, mainly because I can see a list like that going out of date relatively quickly as we introduce new components etc.

Reply:
Yeah that's a good point :+1: